protos/sigstore_bundle.proto

```diff
@@ -67,6 +67,22 @@ message VerificationMaterial {
         TimestampVerificationData timestamp_verification_data = 4;
 }
 
+// DSSESignatureExtension captures the verification materials to be embedded
+// with a Sigstore signature in a DSSE envelope. Specifically, this message is
+// meant to be used for the `ext` field in a signature extension. The `kind` for
+// the Sigstore DSSE extension MUST be
+// application/vnd.dev.sigstore.dsse+json;version=0.1. The certificate chain
```

Comment:
Actually, I'm not sure we need `+json` here as this is never serialized independently.